package.json

```diff
@@ -11,13 +11,13 @@
   "scripts": {
     "start": "yarn build:watch",
     "docs": "update-markdown-jsdoc -f src/random.js --shallow",
-    "build": "babel -s -d dist src",
+    "build": "tsc",
     "build:watch": "babel -w -s -d dist src",
     "prebuild": "yarn run clean",
     "clean": "del dist",
     "prepublishOnly": "yarn run build",
-    "pretest": "yarn run build",
-    "test": "ava -v && standard"
+    "test": "ava -v && standard",
```

Comment:
we may want to remove `standard` in favor of `eslint` w/ `typescript` ala https://github.com/transitive-bullshit/react-static-tweets

Reply:
Done